src/app/src/App.jsx

```diff
@@ -85,12 +85,6 @@ class App extends Component {
                         <main style={appStyles.mainPanelStyle}>
                             <Switch>
                                 <Route
-                                    exact
-                                    path={mainRoute}
-                                    component={MapAndSidebar}
-                                />
-                                <Route
-                                    exact
```

Comment:
`claimFacilityRoute` lost the `exact` here. Was this intentional?

Reply:
This may have been an oversight; I'll double check before merging!

Reply:
I believe this was unintentional. Adding it back did not prevent `/facilities/OARID/claim` from loading as usual. I'll restore the `exact` here.

Reply:
(Conversely, without it `/facilities/OARID/claim/helllo` did match.)